pkgs/development/libraries/libdeflate/default.nix

```diff
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
     description = "Fast DEFLATE/zlib/gzip compressor and decompressor";
     license = licenses.mit;
     homepage = https://github.com/ebiggers/libdeflate;
-    platforms = platforms.linux;
+    platforms = platforms.unix;
```

Comment:
@ryantm Does it make sense that the bot performs this rewrite?

Reply:
It didn't. https://github.com/NixOS/nixpkgs/pull/77128/commits/2170b1eac54259b4e641ca49b6448cb3edf8892f